src/jats/jats-exporter.ts

```diff
@@ -2025,29 +2024,52 @@ export class JATSExporter {
       table.insertBefore(tfoot, tbody as Element)
     }
   }
-
+  private moveBody = (body: HTMLElement) => {
+    const container = body.querySelector(':scope > sec[sec-type="body"]')
+    if (!container) {
+      return
+    }
+    const sections = body.querySelectorAll(
+      ':scope > sec[sec-type="body"] > sec'
+    )
+    sections.forEach((section) => {
+      body.appendChild(section.cloneNode(true))
+    })
+    body.removeChild(container)
+  }
+  private removeBackContainer = (body: HTMLElement) => {
+    const container = body.querySelector(':scope > sec[sec-type="backmatter"]')
```

Comment:
So we're assuming everything has been moved out of this section. Let's check if it is actually empty and log a warning if not.

Reply:
Yeah that's fine.